spamgourmet-clone/scripts/config-certs-provisioning.sh

```diff
@@ -5,7 +5,7 @@ echo '### set up automated provisioning for exim4 and lighttpd'
 echo '##########################################################################'
 source sg-server-config.sh
 
-cd $SCRIPT_BASE_DIR
+cd "$SCRIPT_BASE_DIR" || exit 72
```

Comment:
Why the exit code 72? Is it documented somewhere?

Reply:
Right now it's just a naughty, easily searchable magic number to make it different from other reasons for failure. The LDP defines a range from 64 - 113 as the good ones to use so we at least fit that. 

https://tldp.org/LDP/abs/html/exitcodes.html

we can get around to regularizing this some time soon.